coalahtml/coala_html.py

```diff
@@ -35,7 +35,7 @@ def main():
 
     dir_path = create_dir(os.path.abspath(args.dir))
 
-    if len(list(os.walk(dir_path))) > 0:
+    if len(list(os.walk(dir_path))) > 0:  # pragma: no cover
```

Comment:
Not exactly! I wanted to test if the target directory already has the angular files (copied). But it seems like I overlooked something. For example, consider an empty directory `tests`, the output from `list(os.walk('test'))` is `[('test', [], [])]` which is strange. Thoughts? I assumed it to be 0. So I believe this is a bug!
Now, I have two ways to fix this:
* Still use os.walk approach but change the count to 3 maybe?
* Just check if `dir_path/index.html` exists. This is efficient and neat. Fixing this in separate commit?
@Uran198